modules/nf-core/rgi/cardannotation/meta.yml

```diff
@@ -0,0 +1,43 @@
+name: rgi_cardannotation
+description: Predict antibiotic resistance from protein or nucleotide data
```

Comment:
It doesn't do the prediction does it? Just downloading, no?

Reply:
Yes! I wasn't sure whether to adapt or not (I found other modules where it was not changed). I also find it better to change. Btw. `rgi_cardannotation` is not downloading, just processing the supplied database folder :upside_down_face: At least they have this separate command for it (not like antismash where everything happens in 1 script making it difficult to supply just a downloaded DB).

Reply:
Ah got it, then maybe just state that it preprocesses input database or something :)